src/main.rs

```diff
@@ -136,6 +137,50 @@ pub fn start(mut os_input: Box<dyn OsApi>, opts: Opt) {
     );
     let maybe_layout = opts.layout.map(Layout::new);
 
+    let send_app_instructions_clone = send_app_instructions.clone();
+
+    #[cfg(not(test))]
+    std::panic::set_hook(Box::new(move |info| {
+        let backtrace = Backtrace::new();
```

Comment:
Do you think it could be possible to move this to a different file? Something like `errors.rs`? We'd leave the panic hook here, but call a function that performs all of this?